mage/query-modules/python/export-util.md

```diff
@@ -217,4 +283,112 @@ The `export.json` file should be similar to the one below, except for the
+CREATE (JoyceByers:Character {name:'Joyce Byers', portrayed_by:'Millie Bobby Brown'})
+CREATE (JimHopper:Character {name:'Jim Hopper', portrayed_by:'Millie Bobby Brown'})
+CREATE (MikeWheeler:Character {name:'Mike Wheeler', portrayed_by:'Finn Wolfhard'})
+CREATE (DustinHenderson:Character {name:'Dustin Henderson', portrayed_by:'Gaten Matarazzo'})
+CREATE (LucasSinclair:Character {name:'Lucas Sinclair', portrayed_by:'Caleb McLaughlin'})
+CREATE (NancyWheeler:Character {name:'Nancy Wheeler', portrayed_by:'Natalia Dyer'})
+CREATE (JonathanByers:Character {name:'Jonathan Byers', portrayed_by:'Charlie Heaton'})
+CREATE (WillByers:Character {name:'Will Byers', portrayed_by:'Noah Schnapp'})
+CREATE (SteveHarrington:Character {name:'Steve Harrington', portrayed_by:'Joe Keery'})
+CREATE (MaxMayfield:Character {name:'Max Mayfield', portrayed_by:'Sadie Sink'})
+CREATE (RobinBuckley:Character {name:'Robin Buckley', portrayed_by:'Maya Hawke'})
+CREATE (EricaSinclair:Character {name:'Erica Sinclair', portrayed_by:'Priah Ferguson'})
+CREATE
+(Eleven)-[:ACTED_IN {seasons:[1, 2, 3, 4]}]->(StrangerThings),
+(JoyceByers)-[:ACTED_IN {seasons:[1, 2, 3, 4]}]->(StrangerThings),
+(JimHopper)-[:ACTED_IN {seasons:[1, 2, 3, 4]}]->(StrangerThings),
+(MikeWheeler)-[:ACTED_IN {seasons:[1, 2, 3, 4]}]->(StrangerThings),
+(DustinHenderson)-[:ACTED_IN {seasons:[1, 2, 3, 4]}]->(StrangerThings),
+(LucasSinclair)-[:ACTED_IN {seasons:[1, 2, 3, 4]}]->(StrangerThings),
+(NancyWheeler)-[:ACTED_IN {seasons:[1, 2, 3, 4]}]->(StrangerThings),
+(JonathanByers)-[:ACTED_IN {seasons:[1, 2, 3, 4]}]->(StrangerThings),
+(WillByers)-[:ACTED_IN {seasons:[1, 2, 3, 4]}]->(StrangerThings),
+(SteveHarrington)-[:ACTED_IN {seasons:[1, 2, 3, 4]}]->(StrangerThings),
+(MaxMayfield)-[:ACTED_IN {seasons:[2, 3, 4]}]->(StrangerThings),
+(RobinBuckley)-[:ACTED_IN {seasons:[3, 4]}]->(StrangerThings),
+(EricaSinclair)-[:ACTED_IN {seasons:[2, 3, 4]}]->(StrangerThings);
+```
+</TabItem>
+
+<TabItem value="input_csv">
+
+The image below shows the above data as a graph:
+
+<img src={require('../../data/query-modules/python/export-util/export-util-csv-1.png').default}/>
+    
+</TabItem>
+
+<TabItem value="run_csv">
+
+If you're using **Memgraph with Docker**, the following Cypher query will
+export the database to the `export.csv` file in the
+`/usr/lib/memgraph/query_modules` directory inside the running Docker container.
+
+```cypher
+WITH "MATCH path = (c:Character)-[:ACTED_IN]->(tvshow) RETURN c.name AS name, c.portrayed_by AS portrayed_by, tvshow.title AS title, tvshow.released AS released, tvshow.program_creators AS program_creators" AS query
+CALL export_util.csv_query(query, "/usr/lib/memgraph/query_modules/export.csv", True)
+YIELD file_path, data
+RETURN file_path, data;
+```
+
+If you're using **Memgraph on Ubuntu, Debian, RPM package or WSL**, then the
+following Cypher query will export the database to the `export.csv` file in the
```

Comment:
```suggestion
If you're using **Memgraph on Ubuntu, Debian, RPM package or WSL**, the
following Cypher query will export the database to the `export.csv` file in the
```

Reply:
But how did I not do it 😂  It's haunting me. P.S. read the comment I left in the PR description. kk bye

Reply:
Hm! We could! :) But I can do it so you don't have to worry if you don't have time tomorrow... just write me a changelog msg here, thank you

Reply:
With the `export_util.csv_query()` procedure, you can export query results to a CSV file or as a stream.